models/app.model.js

```diff
@@ -93,3 +93,8 @@ exports.selectUsers = () => {
   })
 }
 
+exports.addCommentCount = (article_id) => {
+  return db.query(`SELECT COUNT(comments.article_id)::int AS comment_count FROM comments WHERE comments.article_id = $1`, [article_id]).then(({ rows }) => {
```

Comment:
Nice use of `::int` to cast the data to the correct data type coming out of the database 🥇